crates/youki/src/workload/mod.rs

```diff
@@ -1,4 +1,6 @@
 pub mod executor;
+#[cfg(feature = "libkrun")]
```

Comment:
Do we really need the feature flag? The libkrun feature has already been protected by the annotation to prevent accidental running. And I don't think this feature will increase the binary and memory footprint size, right?(We should measure that.)

Reply:
I added `libloading` to call (dynamically load) the libkrun library. With the current code, the resulting binary sizes were:

* youki (release):6.2 MB
* youki + libkrun: 6.4 MB
* (youki + WasmEdge): 111 MB

If the differences above aren’t an issue, I’ll remove the features flag.